render-test/main.cpp

```diff
@@ -19,6 +21,21 @@
 #define ANSI_COLOR_LIGHT_GRAY "\x1b[90m"
 #define ANSI_COLOR_RESET      "\x1b[0m"
 
+void* operator new(std::size_t sz) {
```

Comment:
Maybe worth adding?

```c++
void *operator new[](std::size_t size) {
}

void operator delete[](void* ptr) noexcept {
}
```

Reply:
Not needed:
> 2) Called by the array form of new[]-expressions to allocate all storage required for an array (including possible new-expression overhead). The standard library implementation calls version (1)

https://en.cppreference.com/w/cpp/memory/new/operator_new